service/src/main/kotlin/fi/espoo/evaka/pedagogicaldocument/PedagogicalDocumentController.kt

```diff
@@ -200,13 +235,21 @@ private fun Database.Read.findPedagogicalDocumentsByChild(
     return createQuery {
             sql(
                 """
-                SELECT 
+                SELECT
                     pd.id,
                     pd.child_id,
                     pd.description,
                     pd.created,
-                    pd.updated
+                    ce.id AS created_by_id,
+                    ce.name AS created_by_name,
+                    ce.type AS created_by_type,
+                    pd.modified_at,
+                    e.id AS modified_by_id,
+                    e.name AS modified_by_name,
+                    e.type AS modified_by_type
                 FROM pedagogical_document pd
+                LEFT JOIN evaka_user ce ON pd.created_by = ce.id
+                LEFT JOIN evaka_user e ON pd.modified_by = e.id
```

Comment:
If you decide to make `created_by` and `modified_by` as `not null`, then `LEFT` could be removed to communicate that these are always present. Same in all queries.

Reply:
since `created_by` and `modified_by` are now `NOT NULL`, using `JOIN` instead of `LEFT JOIN` would IMH be clearer